virtualmachine/vsphere/vm.go

```diff
@@ -709,7 +709,7 @@ func (vm *VM) AddDisk() error {
 }
 
 // RemoveDisk: removes given list of disks attached to the virtualmachine 'vm'
-// disk.DiskName is the name of the vmdk file for the disk
+// disk.DiskFile is the name of the vmdk file for the disk
 func (vm *VM) RemoveDisk() error {
```

Comment:
This is not part of ur code change but vmMo, err := findVM(vm, dcMo, vm.Name) This line should be out of for loop?

Reply:
Done.